saleor/menu/migrations/0008_menu_json_content_new.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.0.8 on 2018-09-13 13:17
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+import json
+
+
+class Migration(migrations.Migration):
+
+    def populate_data(apps, schema_editor):
+        Menu = apps.get_model('menu', 'Menu')
+        for menu in Menu.objects.all():
+            if isinstance(menu.json_content, str):
```

Comment:
@the-bionic I'm still having an issue with the `Menu` being a str. 
I wonder if we can just change from `if` to `while` in here, wouldn't that solve the problem?

Is it possible that we've stored a `json_content` as a `str`, inside of text-based JSONField? Wouldn't that make it the double nested string?

Reply:
@Pacu2 Yes that did it. I did not realize it was a deeply nested string. The logic for resolving the nested string, though, is a little bit more than changing the ```if``` condition to a ```while``` loop since we transferring the content of a text-based JSONField to a jsonb-based JSONField, before dropping the text-based one.

PR updated again. Thanks.